clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -2585,7 +2630,9 @@ private void pollStatusUntilComplete(
       JobStatusQueryResponse response = ControllerClient.retryableRequest(
           controllerClient,
           pushJobSetting.controllerStatusPollRetries,
-          client -> client.queryOverallJobStatus(topicToMonitor, incrementalPushVersion));
+          client -> client
+              .queryOverallJobStatus(topicToMonitor, incrementalPushVersion, pushJobSetting.targetedRegions));
```

Comment:
Can you test in an integration test if `emergencySourceFabric` is set, but it is not one of the targeted regions, can the push still complete?

As discussed offline, I think it will work fine (with the emergency source region not ingesting any data), but it would be good to confirm it.

Reply:
Yep, you are right.

Wrote a small test case for it. When emergency region is dc-0, and targeted region is dc-1, the jobStatus response will be the following when we query all regions.

```
2023-05-25 17:36:48 - [] INFO [VenicePushJob] [TestNG-method=testTargetedRegionPushJobOverride-1] JobStatusQueryResponse(
version: 2,
status: NOT_CREATED,
statusDetails: ,
extraInfo: {dc-0=NOT_CREATED, dc-1=COMPLETED},
extraDetails: {dc-0=http://adchen-mn1.linkedin.biz:63635 Offline job hasn't been created yet. and Version creation for topic: testTargetedRegionPushJob_1ee5d864cf496_21a1e3f6_v2 got delayed, dc-1=http://adchen-mn1.linkedin.biz:63730 1/1 partitions completed in0 Da Vinci instances.},
ControllerResponse(cluster: venice-cluster0, name: testTargetedRegionPushJob_1ee5d864cf496_21a1e3f6, error: null, errorType: null, exceptionType: null))
```


Since the current implementation is only query targeted region (dc-1), in reality, this job will be reported as COMPLETED.